index.js

```diff
@@ -19,41 +19,63 @@ const zipQuest = async () => {
   return zipFile;
 }
 
-const uploadQuest = async (questId, zipFile) => {
-  console.log(`Uploading ${questId}`);
+const uploadQuest = async (questId, zipFile, onlyValidate = false) => {
```

Comment:
The function is uploadQuest which should upload. It makes less sense to have a `onlyValidate` param. And the same in the Engine. I think we need a `validate` route in the Engine and a `validate` function here